src/cli/index.js

```diff
@@ -2,5 +2,12 @@
  * CLI goes here
  */
 const program = require('commander');
+const packages = require('../../package');
```

Comment:
`packages` => `package`

Reply:
Parsing error: The keyword 'package' is reserved
Hum, I will use packageInfo